docs/cli/config.md

```diff
@@ -6,7 +6,7 @@ Configure CLI options which persist across commands
 * [`celocli config:get`](#celocli-configget)
 * [`celocli config:set`](#celocli-configset)
 
-## `celocli config:get` {#celocli-configget}
```

Comment:
Should these anchors be removed?

Reply:
Yes, even though we should eventually fix the automation one way or the other.